DOCS/canonical-identity-platform-reference-hydra-observability.md

```diff
@@ -2,6 +2,7 @@ The Hydra charmed Operator of the IAM Platform provides integrations to the Cano
 
 In this reference we will provide a technical description of the alert rules, and the dashboards the Hydra operator passes to the components of the COS. We will also showcase the Metrics we use to provide observability features.
 
+> See more: [metrics-endpoint and grafana dashboard integrations](https://charmhub.io/hydra/integrations)
```

Comment:
I think we can say it like on https://github.com/canonical/canonical-identity-platform-docs/blob/main/DOCS/canonical-identity-platform-how-to-integrate-with-oidc-compatible-charms.md?plain=1#L13
```suggestion
> See more: [Charmhub | Hydra > Integrations > `metrics-endpoint`,  `grafana-dashboard`](https://charmhub.io/hydra/integrations)
```